libraries/chaindb/src/pinnable_mapped_file.cpp

```diff
@@ -538,15 +539,29 @@ static std::string print_arch(environment::arch_t arch) {
    return "error";
 }
 
-std::ostream& operator<<(std::ostream& os, const chainbase::environment& dt) {
-   os << std::right << std::setw(17) << "Compiler: " << dt.compiler << '\n';
-   os << std::right << std::setw(17) << "Debug: " << (dt.debug ? "Yes" : "No") << '\n';
-   os << std::right << std::setw(17) << "OS: " << print_os(dt.os) << '\n';
-   os << std::right << std::setw(17) << "Arch: " << print_arch(dt.arch) << '\n';
-   os << std::right << std::setw(17) << "Boost: " << dt.boost_version/100000 << "."
-                                                  << dt.boost_version/100%1000 << "."
-                                                  << dt.boost_version%100 << '\n';
-   return os;
+std::string environment::as_old_string() const {
+   std::stringstream ss;
+   ss << std::right << std::setw(17) << "Compiler: " << compiler.data() << '\n';
+   ss << std::right << std::setw(17) << "Debug: " << (debug ? "Yes" : "No") << '\n';
+   ss << std::right << std::setw(17) << "OS: " << print_os(os) << '\n';
+   ss << std::right << std::setw(17) << "Arch: " << print_arch(arch) << '\n';
+   ss << std::right << std::setw(17) << "Boost: " << boost_version/100000 << "."
+                                                  << boost_version/100%1000 << "."
+                                                  << boost_version%100;
```

Comment:
Any reason to maintain this? Seems like pretty print of JSON of the variant object would be just fine.